app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
     <string name="app_name">AndroidAcademyMsk</string>
+    <string name="name">Avengers: \nEnd Game</string>
+    <string name="pg">13+</string>
+    <string name="tag">Action, Adventure, Fantasy</string>
+    <string name="reviews">125 Reviews</string>
+    <string name="storyline_title">Storyline</string>
+    <string name="storyline">After the devastating events of Avengers: Infinity War, the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to reverse Thanos\' actions and restore balance to the universe.</string>
```

Comment:
don't you want to create a data in MockRepository and fill views from the code?

Reply:
These strings are for creating layouts. I'll use it in xml:
tools:text="@...

Reply:
so all these string will be compiled right into your apk. Place them in the xml file instead.